dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb

```diff
@@ -420,63 +405,47 @@
+    "Let's call the application in the **non-streaming** mode:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 24,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "generations=[[ChatGeneration(text='Image was generated successfully', generation_info={'finish_reason': 'stop', 'logprobs': None}, message=AIMessage(content='Image was generated successfully'))]] llm_output={'token_usage': {}, 'model_name': 'gpt-3.5-turbo'} run=[RunInfo(run_id=UUID('9de1c192-fedc-4a7d-b38d-cd7b4aa221cf'))]\n",
-      "Completion: 'Image was generated successfully'\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "output = llm.generate(messages=[[HumanMessage(content=\"Hello world!\")]])\n",
-    "print(output)\n",
-    "completion = output.generations[0][0].text\n",
-    "print(f\"Completion: {completion!r}\")\n",
-    "assert completion == \"Image was generated successfully\", \"Unexpected completion\""
+    "extra_fields = {\"custom_content\": {\"attachments\": [{\"type\": \"image/png\", \"data\": image_base64}]}}\n",
+    "\n",
+    "try:\n",
+    "  llm.generate(messages=[[HumanMessage(content=\"\", additional_kwargs=extra_fields)]])\n",
+    "\n",
+    "  raise Exception(\"Generation didn't fail\")\n",
+    "except Exception as e:\n",
+    "  assert str(e) == \"Error code: 422 - {'error': {'message': 'No image attachment was found in the last message', 'type': 'runtime_error', 'param': None, 'code': None}}\", \"Unexpected error\""
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the streaming mode:"
+    "Let's call the application in the **streaming** mode:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
+   "execution_count": 25,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "{'content': '', 'additional_kwargs': {}, 'type': 'AIMessageChunk', 'example': False}\n",
-      "{'content': '', 'additional_kwargs': {}, 'type': 'AIMessageChunk', 'example': False}\n",
-      "{'content': 'Image was generated successfully', 'additional_kwargs': {}, 'type': 'AIMessageChunk', 'example': False}\n",
-      "{'content': '', 'additional_kwargs': {}, 'type': 'AIMessageChunk', 'example': False}\n",
-      "Completion: 'Image was generated successfully'\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "output = llm.stream(input=[HumanMessage(content=\"Hello world!\")])\n",
-    "completion = \"\"\n",
-    "for chunk in output:\n",
-    "    print(chunk.dict())\n",
-    "    completion += chunk.content\n",
-    "print(f\"Completion: {completion!r}\")\n",
-    "assert completion == \"Image was generated successfully\", \"Unexpected completion\""
+    "\n",
+    "try:\n",
+    "    output = llm.stream(input=[HumanMessage(content=\"Hello world!\", additional_kwargs=extra_fields)])\n",
```

Comment:
Typo. Here should be an empty content: `content=""` as in other examples in the same article.

Reply:
fixed